language/Records.md

```diff
@@ -20,6 +20,15 @@ Fields of records can be accessed using a dot, followed by the label of the fiel
 ["Functional Programming","JavaScript"]
 ```
 
+Type synonyms for record types are created with the `type` keyword:
+
+```purs
+type Point =
+  { x :: Number
+  , y :: Number
+  }
+```
+
```

Comment:
This is covered later in this documentation, but I suppose it's information worth stating right away. Why not stick with the existing example in this section, though?

> It is common to use type synonyms to for records via the `type` keyword:
>
> ```purs
> type Author = { name :: String, interests :: Array String }
>
> author :: Author
> author = { name: "Phil", interests: ["Functional Programming", "JavaScript"] }
> ```